adapter/parser/src/test/java/org/mobilitydata/gtfsvalidator/parser/GtfsEntityParserTest.java

```diff
@@ -381,7 +381,7 @@ void invalidIntegerValidationShouldGenerateError() {
 
         ErrorNotice notice = result.get(0);
         assertThat(notice, instanceOf(CannotParseIntegerNotice.class));
-        assertEquals("E005", notice.getId());
+        assertEquals(5, notice.getCode());
```

Comment:
If error level is removed from the ID, shouldn't the test also check for the error level?

Reply:
absolutely!